index.js

```diff
@@ -435,11 +435,18 @@
          * Utility function to iterate over the list and call the fn provided
          * on each node, or element, of the list
          *
-         * param {object} fn The function to call on each node of the list
+         * @param {object} fn The function to call on each node of the list
```

Comment:
Oops..good catch.  I do see another mistake I made.  The type of the param for `fn` should be `{function}`.  I can fix that though.